clients/java/signalr/src/test/java/HubConnectionTest.java

```diff
@@ -39,6 +39,76 @@ public void RegisteringMultipleHandlersAndBothGetTriggered() throws Exception {
         assertEquals(2, value.get(), 0);
     }
 
+    @Test
+    public void RemoveHandlerByName() throws Exception {
+
+        AtomicReference<Double> value = new AtomicReference<>(0.0);
```

Comment:
Can you put this all in a `@BeforeTest` to avoid duplication?

Reply:
`@Before` runs before every test method and the method have different set up needs so a one size fits all `@Before` doesn't really work here.

Reply:
Maybe put them in separate classes? Besides that's I don' see much either.

Reply:
Yeah, thought about that but I don't really think it's worth it tbh